src/System.Net.NameResolution/src/project.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
Most likely became unlocked during a merge. I'll set this back to locked.
We'll keep the rest of the contracts unlocked until we submit the _official_ PR to corefx.